src/components/MenuNavigation/MenuNavigation.js

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+import sprite from '../../images/sprite.svg';
+
+export default function MenuNavigation() {
+  return (
+    <nav>
+      <NavLink
+        to="/"
+        className={({ isActive }) => (isActive ? 'active-style' : 'link')}
+      >
+        <svg width="44" height="44">
+          <use href={`${sprite}#icon-home`}></use>
+        </svg>
+        Главная
```

Comment:
We've agreed to write everything in English

Reply:
done